application-templates/starter/webpack.config.prod.js

```diff
@@ -3,10 +3,13 @@ const createWebpackConfigForProduction = require('@commercetools-frontend/mc-scr
 
 const distPath = path.resolve(__dirname, 'dist');
 const entryPoint = path.resolve(__dirname, 'src/index.js');
+const vendorJsFolders = [/node_modules\/omit-empty/];
```

Comment:
Can we find a "better" name for this list? Like this, it looks like you need to pass all the vendor folders or something.

Also, we should somehow document this.

Reply:
Yeah, do you have any suggestions?

As for documenting, we don't have any documentation for mc-scripts so I think we could make that another task right?

Reply:
> Yeah, do you have any suggestions?

Hmm, maybe `untranspiledVendors` or something like it?

> As for documenting, we don't have any documentation for mc-scripts so I think we could make that another task right?

I would be fine to have it in the webpack config for now. Just explain what the purpose of the field is and (e.g. link it to this PR).

Reply:
vendorsToTranspile sounds better to me

I'm not sure what you mean by having it in the webpack config? We can't be sure of every vendor the apps are using...

Reply:
Oh, you mean with just a code comment ?

Reply:
Yes the comment/doc in the webpack config

Reply:
> vendorsToTranspile sounds better to me

👌